.revive.toml

```diff
@@ -23,25 +23,25 @@ enableAllRules = true
 [rule.call-to-gc]
     Disabled = false
 [rule.confusing-naming]
-    Disabled = true
+    Disabled = true # TODO: consider enabling this
 [rule.comment-spacings]
     Disabled = false
 [rule.confusing-results]
```

Comment:
You've disabled this in `containerd.go` , near where I am refactoring this right now. It would be useful to know what this option entails so I can tackle it at the same time.

Reply:
https://github.com/mgechev/revive/blob/master/RULES_DESCRIPTIONS.md#confusing-results

Returning multiple values ​​of the same type without naming them can cause confusion.